include/rest/models/modify_positions_args.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef _modify_positions_args_hpp_
+#define _modify_positions_args_hpp_
+
+#include <models/model.hpp>
+
+namespace disccord
+{
+    namespace rest
+    {
+        namespace models
+        {
+            class modify_positions_args : public disccord::models::model
+            {
+                public:
+                    modify_positions_args(std::vector<std::pair<uint64_t, uint8_t>> id_pos_params);
+                    virtual ~modify_positions_args();
+
+                    std::vector<std::pair<uint64_t, uint8_t>> get_id_pos_params();
+
+                protected:
+                    virtual void encode_to(std::vector<web::json::value>& info) override;
```

Comment:
which is why I added a new overload that encodes to `std::vector<web::json::value>` :) we can encode to object, but now we can encode to array if we want. check in this commit https://github.com/FiniteReality/disccord/pull/21/commits/064cc1b6cb284016972b280f8dffb966dfabe496